inference-engine/samples/benchmark_app/benchmark_app.hpp

```diff
@@ -225,7 +227,7 @@ static void showUsage() {
     std::cout << std::endl << "  device-specific performance options:" << std::endl;
     std::cout << "    -nstreams \"<integer>\"     " << infer_num_streams_message << std::endl;
     std::cout << "    -nthreads \"<integer>\"     " << infer_num_threads_message << std::endl;
-    std::cout << "    -enforcebf16              " << enforce_bf16_message << std::endl;
+    std::cout << "    -enforcebf16=<true/false>     " << enforce_bf16_message << std::endl;
```

Comment:
Update ReadMe as well  and make the same changes for python benchmark_app (readme and parameter description)

Reply:
Done.
I also corrected parsing of enforcebf16 for .py version (by adding choices)

Reply:
Hm, it seems to be an NBC change from interface perspective. I guess it is better to revert this back.

Reply:
Actually is does not affect current interface, choices validation is performed after type conversion, so it is still possible to pass 'y' 'n' '0' '1'. Let me know if i should revert it or not.

Reply:
It is ok for me.